### PR TITLE
get rid of binstar user mention in favor of anaconda user

### PR DESCRIPTION
**Pull request by PeterDSteinberg**
At some point the username in the Dockerfile had changed from binstar to anaconda.  There was a leftover mention of binstar as a user in the Dockerfile that was causing errors in 4 or 5 docker related unittests if docker pull had been run beforehand to pull down the image I built yesterday.

### CODE REVIEW

**Comment by PeterDSteinberg** [comment]
Fixes #213

File: docker/linux-64/Dockerfile [hunk 1]
```diff
@@ -21,7 +21,7 @@ ENV PATH /opt/miniconda/bin:$PATH
 RUN conda config --set always_yes true && \
     conda config --set binstar_upload false && \
     conda config --add channels binstar
-RUN conda install conda-build jinja2 anaconda-client psutil anaconda-build llvm
+RUN conda install conda-build jinja2 anaconda-client psutil binstar-build llvm
```
Comment (dsludwig) [inline docker/linux-64/Dockerfile]: why is this changed back to `binstar-build`?

Comment (PeterDSteinberg) [inline docker/linux-64/Dockerfile]: that was accidental cut and paste error from old PR that was never merged. Will fix.

**Comment by dsludwig** [comment]
lgtm

**Comment by PeterDSteinberg** [comment]
Fixes @213
